lib/dotiw/methods.rb

```diff
@@ -20,15 +20,18 @@ def distance_of_time(seconds, options = {})
     end
 
     def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options = {}, options = {})
-      raise ArgumentError, "nil can't be converted to a Time value" if from_time.nil? || to_time.nil?
+      from_time = normalize_distance_of_time_argument_to_time(from_time)
+      to_time = normalize_distance_of_time_argument_to_time(to_time)
+      from_time, to_time = to_time, from_time if from_time > to_time
```

Comment:
This swap is actually a new feature and changes how the API behaves. What was the outcome without this change before? 

I would prefer not to make this change here and see if we want to add it separately and document it as a breaking change.